crates/router/src/core/payment_link.rs

```diff
@@ -537,13 +660,14 @@ pub async fn get_payment_link_status(
     let payment_link_config = if let Some(pl_config_value) = payment_link.payment_link_config {
         extract_payment_link_config(pl_config_value)?
     } else {
-        admin_types::PaymentLinkConfig {
+        PaymentLinkConfig {
             theme: DEFAULT_BACKGROUND_COLOR.to_string(),
             logo: DEFAULT_MERCHANT_LOGO.to_string(),
             seller_name: merchant_name_from_merchant_account,
             sdk_layout: DEFAULT_SDK_LAYOUT.to_owned(),
             display_sdk_only: DEFAULT_DISPLAY_SDK_ONLY,
             enabled_saved_payment_method: DEFAULT_ENABLE_SAVED_PAYMENT_METHOD,
+            allowed_domains: DEFAULT_ALLOWED_DOMAINS,
         }
```

Comment:
should we make a new structs for `PaymentLinkConfig` as `PaymentLinkOpenConfig` and `PaymentLinkSecureConfig`. As it doesn't make sense to have allowed_domains params inside open_link

Reply:
This is stored in DB as well, we will need to split tables / columns if we want to store different structs